rules/CodingStyle/ClassNameImport/UseImportsTraverser.php

```diff
@@ -71,10 +71,14 @@ private function traverseForType(array $stmts, callable $callable, int $desiredT
                     if ($name === null) {
                         continue;
                     }
+
                     $callable($useUse, $name);
                 }
+
+                return null;
```

Comment:
@staabm here return null on `Use_` and `GroupUse_` ensure no fall to return `NodeTraverser::DONT_TRAVERSE_CURRENT_AND_CHILDREN`